sql/telemetry_derived/clients_daily_histogram_aggregates_v1.sql.py

```diff
@@ -111,6 +111,26 @@ def generate_sql(opts, additional_queries, windowed_clause, select_clause, json_
                 )
                 AND client_id IS NOT NULL),
 
+        sampled_data AS (
+          SELECT
+            *
+          FROM
+            filtered
+          WHERE
+            channel IN ("nightly", "beta")
+            OR (channel = "release" AND os != "Windows")
```

Comment:
You can probably avoid the `union all` section by adding another OR statement here.